sw/device/lib/testing/hmac_testutils.h

```diff
@@ -0,0 +1,85 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OPENTITAN_SW_DEVICE_LIB_TESTING_HMAC_TESTUTILS_H_
+#define OPENTITAN_SW_DEVICE_LIB_TESTING_HMAC_TESTUTILS_H_
+
+#include <assert.h>
+#include <stdint.h>
+
+#include "sw/device/lib/dif/dif_hmac.h"
+
+// Tests need to understand the byte order of the data in the string and
+// the digest values below, as they are laid out for the current processor.
+// RISC-V is little-endian, so the first of these `kHmacTransactionConfig`
+// values is the one we expect to be used, but we include the other for
+// completeness.
+#define HMAC_TESTUTILS_ASSERT_LITTLE_ENDIAN                \
```

Comment:
Even though we should remove it, I'll note that it is poor form to have variable-like macros expand into something with side-effects; make sure to include an empty () in the definition.

Reply:
Removed in the other PR.